src/operator/tensor/elemwise_binary_broadcast_op.h

```diff
@@ -76,6 +77,31 @@ inline bool BinaryBroadcastShape(const nnvm::NodeAttrs& attrs,
   return true;
 }
 
+inline bool BinaryBroadcastStorageTypeCsr(const nnvm::NodeAttrs& attrs,
+                                          const int dev_mask,
+                                          DispatchMode* dispatch_mode,
+                                          std::vector<int>* in_attrs,
+                                          std::vector<int>* out_attrs) {
+  CHECK_EQ(in_attrs->size(), 2U);
+  CHECK_EQ(out_attrs->size(), 1U);
+  const int in1_stype = in_attrs->at(0);
```

Comment:
I'd think left/right-hand side (`lhs`/`rhs`) is a better name compared to `in1`/`in2`

Reply:
done